hack/ukify/main.go

```diff
@@ -298,7 +298,13 @@ func run() error {
 		return err
 	}
 
-	return buildUKI(sdStub, output, sections)
+	if err := buildUKI(sdStub, output, sections); err != nil {
+		return err
+	}
+
+	_, err = sbSign(output)
```

Comment:
are we signing the whole UKI once again? :)

Reply:
yes, that's needed, I forgot earlier :sweat_smile: (the kernel signing is also needed along with this)